src/guides/v2.2/cloud/cdn/fastly-vcl-blocking.md

```diff
@@ -0,0 +1,100 @@
+
+If you want to manage access for both Staging and Production sites, create the Edge ACL with the same name in both environments. The VCL snippet code will apply to both environments.
+
+1. Log in to the Magento Admin.
+1. Navigate to **Stores** > **Settings** > **Configuration** > **Advanced** > **System** > **Fastly Configuration**.
+1. Expand the **Edge ACL** section.
+1. Click **Add ACL** to create a list. For this example, name the list "blocklist".
+1. Enter IP address values in the list. Any client IPs added to this list will be blocked access from the site.
+1. Optionally, select the **Negated** checkbox if needed.
+
+You will reference the Edge ACL by name in your VCL snippet code.
+
+## Create blocklist.json {#vcl}
+
+{:.bs-callout-info}
+This example shows advanced users how to create custom VCL code snippet to configure blocking rules that can be uploaded to the Fastly service using the Fastly API. You can also configure a blocklist or allowlist from the {{ site.data.var.ee }} Admin UI. See [Blocking](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/BLOCKING.md) in the Fastly CDN for Magento 2 module documentation.
+
+After you define the Edge ACL, you can use it to create the VCL snippet to block access to the IP addresses specified in the ACL. You can use the same VCL snippet in both Staging and Production environments, but you must upload the snippet to each environment separately.
+
+Create a `blocklist.json` file with the following VCL code in JSON format:
+
+```json
+{
+  "name": "blocklist",
+  "dynamic": "0",
+  "type": "recv",
+  "priority": "5",
+  "content": "if ( client.ip ~ blocklist) { error 403 \"Forbidden\"; }"
+}
+```
+
+Review the following parameter values and update your code snippet if necessary:
+
+-  `name`: Name for the VCL snippet. For this example, we used the name `blocklist`.
+-  `priority`: Determines the order VCL snippets call. You want to set the priority to 5 to immediately run and check for blocked IP addresses. This priority runs the snippet immediately and before any of the uploaded and default Magento VCL snippets (magentomodule) that have a priority of 50.
+-  `type`: For this VCL, we use `recv`, which places it in the `vcl_recv` subroutine, below the boilerplate VCL and above any objects.
+-  `content`: The snippet of VCL code to run, which checks the client IP address. If the IP is in the Edge ACL, it is blocked from access with a `403 Forbidden` error for the entire website. All other client IP addresses are allowed access.
+
+{:.bs-callout .bs-callout-info}
+The default VCL snippets you uploaded included a prepended name of `magentomodule_` with a priority of 50. For your custom VCL snippets, **do not use the `magentomodule_` name**. Also consider the priority of your custom snippets if they should override the default snippets.
```

Comment:
This callout has some readability problems. I made a suggestion, but you may have a better sense of how to fix both sentences.

```suggestion
 **Do not use the `magentomodule_` name** for your custom VCL Snippets. The default VCL snippets include a prepended name of `magentomodule_` with a priority of 50. We recommend that you change the name and reconsider the priority of your custom snippets.
```